### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wescpy**
added section for Docs API

### CODE REVIEW

**Comment by wescpy** [comment]
Hi, I can't add a reviewer for this change... can someone PTAL? Thx!

File: README.md [hunk 1]
```diff
@@ -22,6 +22,11 @@ Python samples for [G Suite API](https://developers.google.com/gsuite/) docs.
 
 - [Quickstart](https://developers.google.com/classroom/quickstart/python)
 
+### Docs
+
+- [Quickstart](https://developers.google.com/docs/api/quickstart/python)
+- [Mail Merge](docs/mail-merge): Mail merge sample using the Drive, Sheets, and Docs APIs
```
Comment (asrivas) [inline README.md]: Let's wait to add this with the code

**Comment by grant** [comment]
Thanks for the contribution!

File: README.md (hunk 1 above)
Comment (wescpy) [reply README.md]: OK

**Comment by wescpy** [comment]
removed mention of the app until the code gets in

**Pull request merged**